css/style.css

```
@@ -0,0 +1,903 @@
+    }
+    .map iframe {
+        width: 427px
+    }
+}
+
+@media (max-width:1000px) {
+    .container {
+        max-width: 720px
+    }
+    .main-information__text {
+        width: 300px
+    }
+    .main-image__style {
+        width: 310px;
+        -o-object-fit: cover;
+        object-fit: cover
+    }
+    .first-par_text,
+    .first-par_text_second,
+    .second-par_text {
+        line-height: 22px
+    }
+    .featured-works__images {
+        margin-bottom: 100px
+    }
+    .featured-works__images img {
+        width: 350px
+    }
+    .popup__content img {
+        height: 70vh
+    }
+    .form {
+        width: 350px
+    }
+    .contact-contact {
+        width: 350px;
+    }
+    .map iframe {
+        width: 307px
+    }
+}
+
+@media (max-width:800px) {
+    .container {
+        max-width: 540px
+    }
+    .header_text__style {
+        font-size: 30px;
+        line-height: 40px
+    }
+    .main-content {
+        padding-top: 60px
+    }
+    .main-information {
+        flex-direction: column;
+        align-items: center;
+        padding-top: 40px
+    }
+    .main-information__text {
+        width: 500px
+    }
+    .main-image__style {
+        width: 500px;
+        height: 300px;
+        margin-bottom: 40px
+    }
+    .featured-works {
+        padding-top: 80px
+    }
+    .featured-works__images {
+        margin-top: 80px
+    }
+    .featured-works__images img {
+        width: 260px
+    }
+    .popup__content img {
+        height: 50vh
+    }
+    .small-team__images {
+        margin-top: 80px
+    }
+    .contact {
+        padding-top: 50px
+    }
+    .contact-wrapper {
+        flex-direction: column;
+        align-items: center
+    }
+    .contact-form {
+        width: 520px;
+        margin-bottom: 15px
+    }
+    .contact-contact {
+        width: 520px;
+        margin-bottom: 50px
+    }
+    .map iframe {
+        width: 475px
+    }
+}
+
+@media (max-width:600px) {
+    .container {
+        width: 100%;
+        padding: 0 10px
+    }
+    .header {
+        height: 530px
+    }
+    .header_text__style {
+        margin-top: 50px;
+        margin-bottom: 50px
+    }
+    .main-information {
+        padding-top: 40px
+    }
+    .main-information__text {
+        width: 300px
+    }
+    .first-par,
+    .second-par {
+        text-align: center;
+    }
+    .main-image__style {
+        width: 300px;
+        height: 300px;
+        margin-bottom: 40px
+    }
+    .featured-works {
+        padding-top: 50px
+    }
+    .featured-works .featured-works__text {
+        line-height: 35px
+    }
+    .featured-works__images {
+        margin-top: 50px;
+        margin-bottom: 50px
+    }
+    .featured-works__images img {
+        width: 100%
+    }
+    .popup__content img {
+        height: 40vh
+    }
+    .small-team {
+        padding-top: 50px;
+        padding-bottom: 50px
+    }
+    .small-team__images {
+        margin-top: 50px
+    }
+    .team-member-first {

```

.team-member_first ?

---

Thanks 
I fixed all
```suggestion
 .team-member_img1 {
```